pyproject.toml

```diff
@@ -10,3 +10,17 @@ dependencies = [
     "numpy>=1.15",
 ]
 build-backend = "setuptools.build_meta"
+
+[tool.pytest.ini_options]
+markers = [
+    "deeprec: tests for deeprec model",
+    "experimental",
```

Comment:
this is something that we did because some tests were failing, and we didn't want to execute them. See https://github.com/microsoft/recommenders/search?q=%40pytest.mark.experimental. Also it was related to the experimental package https://github.com/microsoft/recommenders/blob/main/setup.py#L91 where we had some libraries that were difficult to install in all python versions.